### ISSUE TITLE
Memory management

### ISSUE THREAD

**Issue by PaulaPerezL** [openclimatefix/Satip#218]
I have some troubles managing memory when reading and saving images. How do you handle it? Do you store each image in its own Zarr, or do you stack them up into one big Zarr?

Thank you very much

**Comment by peterdudfield** [comment]
We do a bit of both,  for our production service we store each image as its own zarr, and then we join then together to make a joint zarr. We seperate non hrv and hrv out. 

We run the service on AWS using these memory limits - https://github.com/openclimatefix/ocf-infrastructure/blob/main/terraform/modules/services/sat/ecs.tf#L4

What memory limits did you have?

@jacobbieker should we put some of this in the readme?

**Comment by peterdudfield** [comment]
Does this make sense @PaulaPerezL ?

**Comment by PaulaPerezL** [comment]
Yes, thank you, it makes sense. What happened to us is that the command "del object" is not working, or at least it does not release the RAM consumed by that object. However, we devised a method to launch each image in an independent process, and by doing so, we are able to clear the RAM after reading the image.

**Comment by peterdudfield** [comment]
Just to let you know, OCF are no longer supporting this repo, as we have moved over to https://github.com/openclimatefix/satellite-consumer

Therefore I will close this issue
